src/background/ui.js

```diff
@@ -11,7 +11,8 @@ const CLOUDFLARE_URL = "https://www.cloudflare.com/";
 const PRIVACY_POLICY_URL = "https://www.mozilla.org/privacy/firefox-private-network";
 const TERMS_AND_CONDITIONS_URL = "https://www.mozilla.org/about/legal/terms/firefox-private-network";
 const GIVE_US_FEEDBACK_URL = "https://qsurvey.mozilla.com/s3/fx-private-network-beta-feedback";
-const BETA_VPN_URL = "https://fpn.firefox.com/vpn";
+const BETA_VPN_FOOTER_URL = "https://fpn.firefox.com/vpn?utm_medium=secure-proxy&utm_source=firefox-browser&utm_campaign=upgrade&utm_content=looking-for-vpn";
+const BETA_VPN_PASSES_EXPIRED_URL = "https://fpn.firefox.com/vpn?utm_medium=secure-proxy&utm_source=firefox-browser&utm_campaign=upgrade&utm_content=passes-expired";
```

Comment:
I'm not a massive fan of blending functionality and tracking but it's fine. I would prefer we had an attribute on the link and a function to handle that attribute to append it to the link. I'm not blocking on that at all though, however I think it's a worthwhile improvement.